chapter_0/Readme.md

```diff
@@ -0,0 +1,16 @@
+Ruby docs
```

Comment:
Please link to the Array, Hash and Chronic docs.  Then find two other interesting Ruby classes to link to their documentation.